CHANGES.md

```diff
@@ -1,5 +1,12 @@
 ## Version 2.1.4 (in development)
-
+* Included normalization of time in case it is called 't' in a dataset instead of 'time' (#956). 
+  This is the case for the following datasets: 
+    * esacci.ICESHEETS.yr.Unspecified.GMB.GRACE-instrument.GRACE.UNSPECIFIED.1-2.greenland_gmb_mass_trends
+    * esacci.ICESHEETS.yr.Unspecified.GMB.GRACE-instrument.GRACE.UNSPECIFIED.1-3.greenland_gmb_mass_trends
+    * esacci.ICESHEETS.yr.Unspecified.GMB.GRACE-instrument.GRACE.UNSPECIFIED.1-5.greenland_gmb_mass_trends
+    * esacci.ICESHEETS.yr.Unspecified.GMB.GRACE-instrument.GRACE.UNSPECIFIED.1-4.greenland_gmb_mass_trends 
```

Comment:
@TonioF I suggested to @AliceBalfanz listing the dataset IDs here so people can immediately see if their datasets are now supported. However, in this case one could do the following to keep it shorter:

```suggestion
   This is the case for all datasets with IDs `esacci.ICESHEETS.yr.Unspecified.GMB.GRACE-instrument.GRACE.UNSPECIFIED.1-*.greenland_gmb_mass_trends`.
```